example/server.js

```diff
@@ -132,6 +132,19 @@ function init (config, callback) {
           },
           {
             method: 'GET',
+            path: '/orgsGetActive',
+            handler: function (request, reply) {
+              request.server.methods.pg.organisations.getActive(function (error, response) { // eslint-disable-line
```

Comment:
I don't think you need eslint-disable-line here, I don't think the length of the line is too long